fuse/ipns/ipns_unix.go

```diff
@@ -238,7 +239,11 @@ func (s *Node) Attr() fuse.Attr {
 			size = 0
 		}
 		if size == 0 {
-			size = s.dagMod.Size()
+			dmsize, err := s.dagMod.Size()
+			if err != nil {
+				log.Error(err)
```

Comment:
what underlying flush, isn't `Attr` only supposed to read? Where's the write to the datastore? (and, that sounds like an error worth surfacing-- )